src/plugins/dam/components/image-asset-fields/index.jsx

```diff
@@ -13,65 +13,68 @@ import TextareaField from '@triniti/cms/components/textarea-field';
 import TextField from '@triniti/cms/components/text-field';
 import './style.scss';
 
-const ImageAssetFields = ({ asset, isEditMode, schemas }) => ([
-  <Card key="asset">
-    <CardHeader>Image Asset</CardHeader>
-    <CardBody indent>
-      <AssetSnippet asset={asset} />
-      <Field
-        name="title"
-        component={TextField}
-        label="Title"
-        placeholder="enter title"
-        readOnly={!isEditMode}
-        size="xlg"
-      />
-      <Field
-        component={PicklistPickerField}
-        isEditMode={isEditMode}
-        label="Credit"
-        name="credit"
-        picklistId="image-asset-credits"
-      />
-      <Field
-        component={DatePickerField}
-        label="Expires At"
-        name="expiresAt"
-        quickSelectOptions={DATE_FIELD_QUICK_SELECT_OPTIONS}
-        readOnly={!isEditMode}
-        showQuickSelect
-        showSetCurrentDateTimeIcon={false}
-        showTime={false}
-      />
-      <Field
-        name="description"
-        label="Description"
-        component={TextareaField}
-        readOnly={!isEditMode}
-      />
-
-    </CardBody>
-  </Card>,
-  schemas.node.hasMixin('triniti:apollo:mixin:has-poll') && (
-    <Card key="poll">
-      <CardHeader>Poll</CardHeader>
-      <CardBody>
-        <FieldArray
-          component={PollPickerField}
+const ImageAssetFields = ({ asset, isEditMode, schemas }) => (
+  <>
```

Comment:
@efjacobson - ha, this is new to me, empty tags instead of fragments?

Reply:
@arunkarnati yep https://reactjs.org/docs/fragments.html#short-syntax